lib/api-group.js

```diff
@@ -23,6 +28,10 @@ class ApiGroup {
     this.requestOptions.cert = options.cert;
     this.requestOptions.key = options.key;
 
+    if (options.hasOwnProperty('insecureSkipTlsVerify')) {
+      this.requestOptions.strictSSL = !options.insecureSkipTlsVerify;
```

Comment:
Yeah we could.  I think it's useful to pull this into the kubernetes-client API (instead of having it as part of the hacky, pass things to `request` API).  For example `insecureSkipTlsVerify` matches the official Go client option (and the `kubectl`) one; and this gives us more flexibility to move off of `request` if we ever want to.